embrace-android-sdk/src/test/resources/breadcrumb_empty.json

```diff
@@ -3,7 +3,7 @@
     "id": "fakeSessionId",
     "st": 160000000000,
     "sn": 1,
-    "ty": "st",
```

Comment:
Wasn't the type validated before?

Reply:
I have altered the default type to be `en` rather than `st` because we no longer send a session start message. `st` will never actually get sent in a payload to the backend as we only send messages with `en` now